docs/pages/management/dynamic-resources/teleport-operator-helm.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Kubernetes Operator in teleport-cluster Helm chart
+description: Deploy the operator alongside your Helm-deployed Teleport Cluster.
+---
+
+This guide explains how to run the Teleport Kubernetes Operator alongside a Teleport cluster
+deployed via the `teleport-cluster` Helm chart.
```

Comment:
Maybe reference "sidecar" somewhere on this page to reduce ambiguity?

Reply:
Since v15 it's not sidecar anymore :)